src/resmom_win/start_exec.c

```diff
@@ -2211,28 +2256,16 @@ finish_exec(job *pjob)
 					return;
 				}
 
-				/* numenv is the total # of */
-				/* slots in env_array including */
-				/*  unused slots */
-				/* curenv is the total # of used */
-				/* slots */
```

Comment:
Why remove this comment? Is it inaccurate?

Reply:
It no longer applies to the updated code, it referred to a previous line that goes "curenv = numenv".